docs/user/reference/openapi.yaml

```diff
@@ -0,0 +1,217 @@
+components:
+  schemas:
+    DeviceModel:
+      additionalProperties: false
+      description: Representation of a device
+      properties:
+        name:
+          description: Name of the device
+          title: Name
+          type: string
+        protocols:
+          description: Protocols that a device conforms to, indicating its capabilities
+          items:
+            type: string
+          title: Protocols
+          type: array
+      required:
+      - name
+      - protocols
+      title: DeviceModel
+      type: object
+    DeviceResponse:
+      additionalProperties: false
+      description: Response to a query for devices
+      properties:
+        devices:
+          description: Devices available to use in plans
+          items:
+            $ref: '#/components/schemas/DeviceModel'
+          title: Devices
+          type: array
+      required:
+      - devices
+      title: DeviceResponse
+      type: object
+    HTTPValidationError:
+      properties:
+        detail:
+          items:
+            $ref: '#/components/schemas/ValidationError'
+          title: Detail
+          type: array
+      title: HTTPValidationError
+      type: object
+    PlanModel:
+      additionalProperties: false
+      description: Representation of a plan
+      properties:
+        name:
+          description: Name of the plan
+          title: Name
+          type: string
+      required:
+      - name
+      title: PlanModel
+      type: object
+    PlanResponse:
+      additionalProperties: false
+      description: Response to a query for plans
+      properties:
+        plans:
+          description: Plans available to use by a worker
+          items:
+            $ref: '#/components/schemas/PlanModel'
+          title: Plans
+          type: array
+      required:
+      - plans
+      title: PlanResponse
+      type: object
+    TaskResponse:
+      additionalProperties: false
+      description: Acknowledgement that a task has started, includes its ID
+      properties:
+        taskName:
+          description: Unique identifier for the task
+          title: Taskname
+          type: string
+      required:
+      - taskName
+      title: TaskResponse
+      type: object
+    ValidationError:
+      properties:
+        loc:
+          items:
+            anyOf:
+            - type: string
+            - type: integer
+          title: Location
+          type: array
+        msg:
+          title: Message
+          type: string
+        type:
+          title: Error Type
+          type: string
+      required:
+      - loc
+      - msg
+      - type
+      title: ValidationError
+      type: object
+info:
+  title: FastAPI
```

Comment:
This should have a better Title;  I would suggest just 'BlueAPI' as the fact that it's and OpenAPI spec tells us it's REST already

Reply:
Have updated.